cellrank/kernels/_monge_kernel.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any, Tuple, Union, Literal, Callable, Optional, Sequence
+
+from moscot.problems.time import TemporalNeuralProblem
+
+from cellrank._utils._docs import d, inject_docs
+from scvelo.preprocessing.moments import get_moments
+from cellrank.kernels._velocity_kernel import VelocityKernel
+from cellrank.kernels.utils._similarity import Similarity
+from cellrank.kernels.utils._velocity_model import BackwardMode, VelocityModel
+
+import numpy as np
+
+__all__ = ["MongeKernel"]
+
+_ATTR_OPTIONS = Literal["obsm", "layers"]
```

Comment:
This is used only 2x (or 3x), but I think it's fair to use this type-hint in place rather than defining an alias.